projects/vdk-plugins/vdk-meta-jobs/tests/test_tracking_job_executor.py

```diff
@@ -25,7 +25,7 @@ def test_get_latest_available_execution_id():
     test_executor = MagicMock(spec=RemoteDataJobExecutor)
     test_executor.job_executions_list.side_effect = [[test_execution]]
 
-    tracking_executor = TrackingDataJobExecutor(test_executor)
+    tracking_executor = TrackingDataJobExecutor(test_executor, 40)
```

Comment:
Nit: Export this magic number to a constant if possible.

Reply:
It's fine as is for the purposes of the test, we have other tests where numbers are included in this way.